05_syntax_lab/01.py

```diff
@@ -1,5 +1,7 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+############### Part 1
+print('Please enter 10 numbers in one raw, separated by a space')
+numbers = input()
+print('These are the numbers:')
+print(numbers)
+print('The biggest value is:')
+print(max(numbers))
```

Comment:
I am working with python 3.3.2 (see “print” and of course “input”).
This is the installation on my PC.
10X

From: Gabor Szabo [mailto:notifications@github.com]
Sent: Tuesday, July 05, 2016 11:37 AM
To: ynonp/python-examples-verint-2016-07
Cc: Rosenthal, Hanan (Embedded); Author
Subject: Re: [ynonp/python-examples-verint-2016-07] ToCode Solution: תרגול תחביר (#14)


In 05_syntax_lab/01.py<https://github.com/ynonp/python-examples-verint-2016-07/pull/14#discussion_r69525035>:

> @@ -1,5 +1,7 @@

> -"""

> -Write a program that reads 10 numbers from

> -the user and prints the largest one

> -"""

> -

> +############### Part 1

> +print('Please enter 10 numbers in one raw, separated by a space')

> +numbers = input()

> +print('These are the numbers:')

> +print(numbers)

> +print('The biggest value is:')

> +print(max(numbers))

Using input() in Python 2 is dangerous. Please use raw_input instead of it.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ynonp/python-examples-verint-2016-07/pull/14/files/7d8ebcf1c1fd7e7419e80c5b8f410dd6dc1218d5#r69525035>, or mute the thread<https://github.com/notifications/unsubscribe/ATVHXq7ygSaGbmFuwq_a6AfDJJF_O9Neks5qShelgaJpZM4JE3pq>.

-- 
This electronic message may contain proprietary and confidential 
information of Verint Systems Inc., its affiliates and/or subsidiaries.
The information is intended to be for the use of the individual(s) or
entity(ies) named above. If you are not the intended recipient (or 
authorized to receive this e-mail for the intended recipient), you may not 
use, copy, disclose or distribute to anyone this message or any information 
contained in this message. If you have received this electronic message in 
error, please notify us by replying to this e-mail.